app/styles/site-views/form-view.scss

```diff
@@ -0,0 +1,22 @@
+
+
+.centerdiv-form-wrapper {
+  margin-top: 20px;
+  margin-bottom: 40px;
+}
+
+.centereddiv-form {
+  height: auto;
+  width:auto;
+  // background-color: #000;
+}
```

Comment:
delete the property if it won't be used. don't just comment it out.

Reply:
removed